app/views/loans/newloanaccount.html

```diff
@@ -929,7 +929,7 @@ <h3 ng-show="loanaccountinfo">{{ 'label.heading.savingsLinkage' | translate}}</h
                         </tr>
                         <tr ng-hide="response.uiDisplayConfigurations.loanAccount.isHiddenField.interestType">
                             <td colspan="6">{{ 'label.input.interestmethod' | translate }}</td>
-                            <td>{{loandetails.interestValue}}</td>
+                            <td>{{loandetails.interestValue}}<span ng-if="!loandetails.interestValue">{{ 'label.input.interestmethod.default' | translate }}</span></td>
```

Comment:
This is not the correct way to show these values. Please ensure that if there is a default value it is set in the loandetails object so that it is visible during the preview. Remove the unnecessary span with ngIf and translations.